packages/connect-wallet-modal/src/components/ReefKnotWalletsModal/walletsModalConfig.ts

```diff
@@ -0,0 +1,57 @@
+import {
```

Comment:
This import should be below '@reef-knot/wallets-list'

Reply:
Done